test/python/circuit/library/test_adders.py

```diff
@@ -60,17 +60,22 @@ def assertAdditionIsCorrect(self, num_state_qubits, adder, inplace):
 
     @data(
         (3, RippleCarryAdder, True),
-        (5, RippleCarryAdder, True)
+        (5, RippleCarryAdder, True),
+        (3, QFTAdder, True),
+        (5, QFTAdder, True),
+        (3, QFTAdder, True, True),
+        (5, QFTAdder, True, True)
```

Comment:
It seems like it might get a little messy if we come up with new adders that might have other arguments. Also, it is not that straightforward to read. What do you think about changing the structure to something like
```python
@data(
    (3, QFTAdder(3, modular=True)),
    (n, OtherAdder(n, other_options))
 )
 @unpack
 def test_summation(self, num_state_qubits, adder):
    # test here
```

Reply:
Actually... that doesn't work with the way `testAdditionIsCorrect` works now. Ok then we can leave it as is 👍🏻